test-cypress/steps/graphql/graphql-playground-steps.js

```diff
@@ -68,6 +68,10 @@ export class GraphqlPlaygroundSteps {
         return this.getExecuteButton().click();
     }
 
+    static abortQuery() {
```

Comment:
BTW, I just now noticed that you've added another steps class for the playground called GraphiqlPlaygroundSteps . Do you mind removing it?

Reply:
Removed